### ISSUE TITLE
Support of latest app-blueprint

### ISSUE THREAD

**Issue by evoactivity** [patricklx/ember-vite-hmr#58]
I'm trying this out with the latest app-blueprint which includes work to stop rewriting the app.

I see vite is hot reloading the modules, but I don't see my component reload in the browser.

All I have done is `ember install ember-vite-hmr` and added `hmr()` to my vite config.

```js
import { defineConfig } from "vite";
import {
  resolver,
  hbs,
  scripts,
  templateTag,
  optimizeDeps,
  compatPrebuild,
  assets,
  contentFor,
} from "@embroider/vite";
import { babel } from "@rollup/plugin-babel";
import { hmr } from "ember-vite-hmr";

const extensions = [
  ".mjs",
  ".gjs",
  ".js",
  ".mts",
  ".gts",
  ".ts",
  ".hbs",
  ".json",
];

export default defineConfig(({ mode }) => {
  return {
    resolve: {
      extensions,
    },
    plugins: [
      hbs(),
      templateTag(),
      scripts(),
      resolver(),
      compatPrebuild(),
      assets(),
      contentFor(),

      babel({
        babelHelpers: "runtime",
        extensions,
      }),
      hmr(),
    ],
    optimizeDeps: optimizeDeps(),
    server: {
      port: 4200,
    },
    build: {
      outDir: "dist",
      rollupOptions: {
        input: {
          main: "index.html",
          ...(shouldBuildTests(mode)
            ? { tests: "tests/index.html" }
            : undefined),
        },
      },
    },
  };
});

function shouldBuildTests(mode) {
  return mode !== "production" || process.env.FORCE_BUILD_TESTS;
}
```

**Comment by patricklx** [comment]
fixed in release 1.4.16

**Comment by evoactivity** [comment]
I had to remove `@fs` to make it work

```diff
- id = '/@fs' + id;
+ id = '/' + id;
```
